### PR TITLE
Feature/flex mixins

### PR DESCRIPTION
**Pull request by JSerhii**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ Example:
 - [Typography](./docs/typography.md)
 - [Units](./docs/units.md)
 - [Utils](./docs/utils.md)
+- [Flex](./docs/flex.md)
```
Comment (Nerdy-Girl) [inline README.md]: alphabetical order please 🙏

File: docs/flex.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
+# _flex.scss
+_File includes mixins which helps use flex features in easiest way_
```
Comment (Nerdy-Girl) [inline docs/flex.md]: *which help using

File: docs/flex.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+- `$cross` - defines how flex items are laid out along the cross axis
+- `$main` - defines the alignment along the main axis
+- `$wrap` - defines how flex items will wrap onto multiple lines
+
+### Usage: 
+
+#### Case 1
+Assign property `display: flex;` to the class without any extra properties.
+
+```
```
Comment (Nerdy-Girl) [inline docs/flex.md]: @ArekRado will cry for it -> ```scss where needed please :)

File: src/_base.scss [hunk 4]
```diff
@@ -4,4 +4,5 @@
 @import "fonts";
 @import "typography";
 @import "units";
-@import "utils";
\ No newline at end of file
+@import "utils";
+@import "flex";
```
Comment (Nerdy-Girl) [inline src/_base.scss]: alphabetical order please 🙏

**Review by Nerdy-Girl — CHANGES_REQUESTED** [review]

File: src/_base.scss (hunk 4 above)
Comment (JSerhii) [reply src/_base.scss]: fixed

File: docs/flex.md (hunk 3 above)
Comment (JSerhii) [reply docs/flex.md]: fixed

File: docs/flex.md (hunk 2 above)
Comment (JSerhii) [reply docs/flex.md]: fixed

File: README.md (hunk 1 above)
Comment (JSerhii) [reply README.md]: fixed

**Review by Nerdy-Girl — APPROVED** [review]

**Review by ArekRado — APPROVED** [review]

**Pull request merged**